pyproject.toml

```diff
@@ -180,7 +180,7 @@ venv = ".venv"
 
 [tool.codespell]
 skip = '.git,*.lock'
-ignore-words-list = 'EHR'
+ignore-words-list = 'EHR, AKS, Appen'
```

Comment:
wait so which files were these from? i'm assuming the entire file ignore you added to the messages file didn't work, but I wonder if we could add a file ignore here instead that would work rather than adding words to ignore?

Reply:
i tried adding a file-ignore to the skip, but wasnt sure the best way to do so. it's under `web_search_agent/messages.py`

Reply:
does the file ignore not cover these specific words?

Reply:
file ignore doesnt seem to ignore file, not sure why

Reply:
Try adding the skip to the `pyproject.toml`:

```toml
[tool.codespell]
skip=[".git", "*.lock", "examples/cookbook/agents/web_search_agent/messages.py"]
```

I'm not certain the list format works, so may just need to add it to the comma separated list 🤷

Reply:
this works